portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -20,13 +20,46 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import com.google.gson.Gson;
+import java.util.ArrayList;
+
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+    public class Message{
+        String sender, message, recipient;
+        public Message(String sndr, String msg, String rcpnt){
+            sender = sndr;
+            message = msg;
+            recipient = rcpnt;
+        }
+    }
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        //response.setContentType("text/html;");
```

Comment:
With version system in place (e.g., Git here), we usually just remove unused code rather than commenting them out because this lowers the burden for readers, unless this code serves a particular purpose for understanding the code (which doesn't seem to be the case here).